semgrep-core/src/engine/Match_tainting_rules.ml

```diff
@@ -159,13 +160,9 @@ let taint_config_of_rule default_config equivs file ast_and_errors
            if not_conflicting then
              if
                not
-                 (List.exists
-                    (fun rng' ->
-                      rng'.Range_with_metavars.r = rng.Range_with_metavars.r)
-                    sinks_ranges
+                 (List.exists (fun rng' -> rng'.RM.r = rng.RM.r) sinks_ranges
                  || List.exists
-                      (fun rng' ->
-                        rng'.Range_with_metavars.r = rng.Range_with_metavars.r)
+                      (fun rng' -> rng'.RM.r = rng.RM.r)
```

Comment:
I know you're just making a small change here, but I think using the alias for `Range_with_metavars` really makes this alphabet soup. Can we use a more descriptive name for `rng`?

Reply:
I replaced it with `range`.